apps/server/src/shared/domain/service/lesson-copy.service.ts

```diff
@@ -8,21 +8,30 @@ export type LessonCopyParams = {
 	user: User;
 };
 
-export type LessonCopyResponse = {
-	// copy: Lesson;
-	status: CopyStatus;
-};
-
 @Injectable()
 export class LessonCopyService {
-	copyLesson(params: LessonCopyParams): LessonCopyResponse {
-		const result = {
-			status: {
-				title: params.originalLesson.name,
-				type: CopyElementType.LESSON,
-				status: CopyStatusEnum.NOT_IMPLEMENTED,
-			},
+	copyLesson(params: LessonCopyParams): CopyStatus {
+		const copy = new Lesson({
+			course: params.destinationCourse,
+			hidden: params.originalLesson.hidden,
+			name: `${params.originalLesson.name} (copy)`, // TODO remove/translate (copy)?
+			position: params.originalLesson.position,
+		});
+
+		const status: CopyStatus = {
+			title: copy.name,
+			type: CopyElementType.LESSON,
+			status: CopyStatusEnum.PARTIAL,
```

Comment:
long term partial should only be used when at least one element is failed or notimplemented. maybe add a temporary dummy element (eg "content") to communicate that things are missing?